@here/harp-datasource-protocol/lib/DecodedTile.ts

```diff
@@ -62,6 +63,12 @@ export interface DecodedTile {
      * @see [[CopyrightInfo]]
      */
     copyrightHolderIds?: string[];
+
+    /**
+     * List of {@link @here/harp-geoutils#TileKey}s representing {@link @here/harp-mapview#Tile}s
+     * that have geometry covering this `Tile`.
+     */
+    dependencies?: TileKey[];
```

Comment:
you cannot have TIleKey in DecodedTile because DecodedTile may be transferred from the webworker to the main app so it cannot contain object instances. You should serialize the list of tile keys to morton codes or strings here,
and then convert them to TIleKey when initializing the `Tile`.